.gitignore

```diff
@@ -168,3 +168,7 @@ cython_debug/
 
 # PyPI configuration file
 .pypirc
+controller/ClassicMan.blend1
+controller/ClassicMan.blend
```

Comment:
nah this is not necessary. We will use this controller on any model character so we can't expect everyone to have the same filename

Reply:
still an issue